Jenkinsfile

```diff
@@ -778,7 +778,7 @@ def generateServiceName(uniqueID, servicePrefix = '') {
   if (getParam('USE_LATEST_RELEASE', false) || getParam('GCLOUD_URL')) {
     return "${uniqueID}-dot-${PROJECT_ID}.appspot.com"
   }
-  return "${servicePrefix}testing-dot-${PROJECT_ID}.appspot.com"
+  return "${servicePrefix}-${uniqueID}-dot-${PROJECT_ID}.appspot.com"
```

Comment:
This function is only used for GCE test.   GKE is here

https://github.com/cloudendpoints/esp/blob/master/script/e2e-kube.sh#L58